### ISSUE TITLE
Enable edit collaborator Permission tables/descriptions

### ISSUE THREAD

**Issue by pengchenglai** [openforis/fra-platform#2528]
@minotogna 
The permission button which allow NC to edit the accessibility of the reporting tables for collaborater is missing.
![2](https://user-images.githubusercontent.com/99216009/230533191-7be6605a-e8cb-4210-9f1a-652c51c09e62.JPG)

![1](https://user-images.githubusercontent.com/99216009/230532895-a986f2f5-9c25-4c87-9b5f-80581c3bde53.JPG)
